node/src/neighborhood/gossip_acceptor.rs

```diff
@@ -2955,14 +3171,13 @@ mod tests {
             .add_half_neighbor_key(existing_node_key.clone())
             .unwrap();
         root_node.increment_version();
+        root_node.metadata.last_update = dest_db.root().metadata.last_update;
         root_node.resign();
         assert_eq!(&root_node, dest_db.root());
-        assert_node_records_eq(
-            dest_db.node_by_key_mut(debut_node.public_key()).unwrap(),
-            &debut_node,
-            before,
-            after,
-        )
+        let reference_node = dest_db.node_by_key_mut(debut_node.public_key()).unwrap();
```

Comment:
Taky by nemuselo byt ..`_key_mut`

A v dalsich pripadech v tomto souboru take...ale uz to nebudu uvadet...